components/common/src/ui.rs

```diff
@@ -43,6 +44,169 @@ pub const NONINTERACTIVE_ENVVAR: &str = "HAB_NONINTERACTIVE";
+            UISymbolStyle::Ascii => "ascii",
+        };
+        write!(f, "{}", msg)
+    }
+}
+
+#[derive(Clone, Copy)]
+pub enum UISymbol {
+    UpArrow,
+    FingerPoint,
+    CheckMark,
+    BoxedCheckMark,
+    Omega,
+    BoxedX,
+    RightArrow,
+    Cloud,
+    DownArrow,
+    Elipses,
+    DottedTriangle,
+    RightShift,
+    Star,
+    SlashedZero,
+    ErrorX,
+}
+
+impl UISymbol {
+    pub fn to_str(&self) -> &str {
+        let style = if let Ok(s) = env::var(SYMBOL_STYLE_ENVVAR) {
+            match UISymbolStyle::from_str(&s) {
+                Ok(style) => style,
+                Err(e) => {
+                    eprintln!("{}\nSetting UISymbolStyle to {}",
+                              e,
+                              UISymbolStyle::default());
+                    UISymbolStyle::default()
+                }
+            }
+        } else {
+            UISymbolStyle::default()
+        };
+
+        match style {
+            UISymbolStyle::Ascii => {
+                match *self {
+                    UISymbol::UpArrow => "/^\\",
+                    UISymbol::FingerPoint => "-->",
+                    UISymbol::CheckMark => "[x]",
+                    UISymbol::BoxedCheckMark => "[#]",
+                    UISymbol::Omega => "-->",
+                    UISymbol::BoxedX => "-X-",
+                    UISymbol::RightArrow => "-->",
+                    UISymbol::Cloud => "-->",
+                    UISymbol::DownArrow => "  >",
+                    UISymbol::Elipses => "...",
+                    UISymbol::DottedTriangle => "???",
+                    UISymbol::RightShift => " >>",
+                    UISymbol::Star => "***",
+                    UISymbol::SlashedZero => "  0",
+                    UISymbol::ErrorX => "XXX",
+                }
+            }
+            UISymbolStyle::Limited => {
+                match *self {
+                    UISymbol::UpArrow => "↑",
+                    UISymbol::FingerPoint => "→",
+                    UISymbol::CheckMark => "√",
+                    UISymbol::BoxedCheckMark => "⌂",
+                    UISymbol::Omega => "Ω",
+                    UISymbol::BoxedX => "░",
+                    UISymbol::RightArrow => "→",
+                    UISymbol::Cloud => "⌂",
+                    UISymbol::DownArrow => "↓",
+                    UISymbol::Elipses => "…",
+                    UISymbol::DottedTriangle => "‼",
+                    UISymbol::RightShift => "»",
+                    UISymbol::Star => "≡",
+                    UISymbol::SlashedZero => "Ø",
+                    UISymbol::ErrorX => "XXX",
+                }
+            }
+            UISymbolStyle::Full => {
+                match *self {
+                    UISymbol::UpArrow => "↑",
+                    UISymbol::FingerPoint => "☛",
+                    UISymbol::CheckMark => "√",
+                    UISymbol::BoxedCheckMark => "☑",
+                    UISymbol::Omega => "Ω",
```

Comment:
I think we should actually get rid of this one. The only place it's used is in a "Creating" message, and I think that would be more consistent to use `::FingerPoint` (or `::InProgress`) since that's consistently used for all the "-ing" statuses.